charts/external-dns/CHANGELOG.md

```diff
@@ -20,6 +20,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed support for gateway-grpcroute, gateway-tlsroute, gateway-tcproute, gateway-udproute
+  https://github.com/kubernetes-sigs/external-dns/issues/3845 
+
+
+## [v1.14.3]
+
+### Fixed
+
```

Comment:
```suggestion
- Fixed support for gateway-grpcroute, gateway-tlsroute, gateway-tcproute, gateway-udproute
  https://github.com/kubernetes-sigs/external-dns/issues/3845 
```
You only need to add the line item, and ideally below the previous one but I couldn't get the suggestion to cover the un-changed line.